src/test/scala/cromwell/engine/db/slick/SlickDataAccessSpec.scala

```diff
@@ -262,9 +262,12 @@ class SlickDataAccessSpec extends FlatSpec with Matchers with ScalaFutures {
             new Call(callAlias, "call.name", task, Set.empty[FullyQualifiedName], Map.empty, None)
           }
 
+        val callKey = ExecutionDatabaseKey(call.fullyQualifiedName, None)
+        val shardKey = ExecutionDatabaseKey(call.fullyQualifiedName, Option(0))
```

Comment:
If we do that for those two then it could/should also be done for the callFqn, symbolFqn... used everywhere in this spec no ?
```scala
val callFqn = "call.fully.qualified.scope"
val symbolFqn = "symbol.fully.qualified.scope"
```